src/servers/graphql-old-server.ts

```diff
@@ -374,7 +379,24 @@ const resolvers = {
 
         return status.value
       },
-      getFee: async ({ amount, invoice }) => wallet.getLightningFee({ amount, invoice }),
+      getFee: async ({ amount, invoice }) => {
+        let feeSatAmount: Satoshis | ApplicationError
+        if (amount && amount > 0) {
+          feeSatAmount = await lnNoAmountInvoiceFeeProbe({
+            amount,
+            paymentRequest: invoice,
+          })
+          if (!(feeSatAmount instanceof Error)) return feeSatAmount
+          if (!(feeSatAmount instanceof LnPaymentRequestZeroAmountRequiredError))
+            throw mapError(feeSatAmount)
```

Comment:
not sure if I am missing something but maybe is cleaner to have something similar to `lnInvoiceFeeProbe` 
```
if (feeSatAmount instanceof Error) throw mapError(feeSatAmount)
return feeSatAmount
```

Reply:
I believe with the current code we want the function to continue on if it's an error that isn't a `LnPaymentRequestZeroAmountRequiredError` error.

With the suggested change the function would stop here on all errors.